### PR TITLE
[GP5-2] As a non-registered user, I want to be able to register as a user

### PR DESCRIPTION
**Pull request by RaihanAdity**

### CODE REVIEW

File: projects/client/src/pages/Register.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                <Card maxWidth='300px'>
+                    <CardBody>
+                        <FormControl>
+                            <FormLabel size='sm'>
+                                First Name
+                            </FormLabel>
+                            <Input type='text' bg='white' borderColor='#d8dee4' size='sm' borderRadius='6px' placeholder="First Name" required ref={inputFirstName} onChange={(e) => validateFirstName(e.target.value)} />
+                            <div className=" text-red-700 text-sm font-semibold font-ibmFont">
+                                {errFirstName ? errFirstName : null}
+                                {console.log(errFirstName)}
```
Comment (haikal1223) [inline projects/client/src/pages/Register.jsx:101]: remove console

File: projects/client/src/components/navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default function Navbar() {
+    return <div></div>;
+  }
+
```
Comment (haikal1223) [inline projects/client/src/components/navbar.jsx:4]: remove code

File: projects/client/src/pages/newPassword.jsx [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export default function NewPassword() {
+    return <div></div>;
+  }
+
```
Comment (haikal1223) [inline projects/client/src/pages/newPassword.jsx:4]: remove code

**Review by rohmatsetiawanmta — APPROVED** [review]

File: projects/client/src/components/Button/card.jsx
Comment (doomsdoge) [inline projects/client/src/components/Button/card.jsx:1]: Delete, pushed via GP5-1

File: projects/client/src/pages/SignUp.jsx
Comment (doomsdoge) [inline projects/client/src/pages/SignUp.jsx:1]: Delete

File: projects/server/sequelize/migrations/20230308174242-create-user-address.js
Comment (doomsdoge) [inline projects/server/sequelize/migrations/20230308174242-create-user-address.js:1]: Delete

**Review by doomsdoge — APPROVED** [review]

**Review by BarsMoxxi — APPROVED** [review]

**Review by haikal1223 — APPROVED** [review]

**Pull request merged**